spyderlib/plugins/editor.py

```diff
@@ -105,12 +105,12 @@ def setup_page(self):
                                     text=_("Text and margin font style"),
                                     fontfilters=QFontComboBox.MonospacedFonts)
         newcb = self.create_checkbox
-        fpsorting_box = newcb(_("Sort files according to full path"),
-                              'fullpath_sorting')
+#        fpsorting_box = newcb(_("Sort files according to full path"),
+#                              'fullpath_sorting')

```

Comment:
So this was a current option? Then we should remove in this PR because our decision is to order files according to their opening order.